bg_atlasapi/__init__.py

```diff
@@ -1,5 +1,13 @@
-__author__ = """brainglobe"""
-__version__ = "1.0.2"
+from importlib.metadata import PackageNotFoundError, version
+
+try:
+    __version__ = version("bg-atlasapi")
+    del version
+except PackageNotFoundError:
+    # package is not installed
+    pass
+
+__author__ = "BrainGlobe Developers"
```

Comment:
```suggestion
from importlib.metadata import PackageNotFoundError, metadata

try:
    __version__ = metadata("bg-atlasapi")["Version"]
    __author__ = metadata("bg-atlasapi")["Author"]
    del metadata
except PackageNotFoundError:
    # package is not installed
    pass
```

If you want to avoid having to update the authors (and other info that is pulled from pyproject too)

Reply:
Cool, thanks.